docs/source/tutorial/dags.rst

```diff
@@ -71,8 +71,10 @@ overhead by using :meth:`.PyDiGraph.add_parent` and
 :meth:`.PyDiGraph.add_child` as both add a new node and edge simultaneously and
 neither can introduce a cycle.
 
-.. [#] A trail is a sequence of nodes and edges on the graph where each edge
-   is distinct in the walk.
+.. note::
```

Comment:
I'm not sure we want to use a note directive here as it will render as a standalone box that says note. Maybe having a subsubsection header like:

```suggestion
Footnotes:
^^^^^^^^^^
```

Will clearly mark it as separate but avoid highlighting it in a new box